build-logic/src/main/kotlin/geyser.modrinth-uploading-conventions.gradle.kts

```diff
@@ -12,7 +12,7 @@ modrinth {
     versionNumber.set(projectVersion(project))
     versionType.set("beta")
     changelog.set(System.getenv("CHANGELOG") ?: "")
-    gameVersions.addAll("1.21", libs.minecraft.get().version as String)
+    gameVersions.addAll("1.21.2", libs.minecraft.get().version as String)
```

Comment:
1.21.3 should be added.

Reply:
that's already being done by the libs' Minecraft version reference

Reply:
Project maintainer reading comprehension challenge FAILED